docs/source/integrations/mobile/ios-fairplay-sdk.md

```diff
@@ -202,6 +202,10 @@ catch {
 
 ## Release notes
 
+### v0.0.1 (build 105) on 24/09/2019
```

Comment:
Clarifying no version number change on slack:
https://vualtoteam.slack.com/archives/GH5535346/p1569402960002800

Reply:
this is correct.